src/coreclr/jit/fgbasic.cpp

```diff
@@ -855,9 +893,8 @@ void Compiler::fgFindJumpTargets(const BYTE* codeAddr, IL_OFFSET codeSize, Fixed
     const bool  isPreJit               = opts.jitFlags->IsSet(JitFlags::JIT_FLAG_PREJIT);
     const bool  resolveTokens          = makeInlineObservations && (isTier1 || isPreJit);
     unsigned    retBlocks              = 0;
-    unsigned    intrinsicCalls         = 0;
     int         prefixFlags            = 0;
-    int         value                  = 0;
+    bool        conservativeInliner    = JitConfig.JitConservativeInliner();
```

Comment:
You should make the fidelity of the stack model be a property of the inline policy.

Reply:
Done